docs/pages/components/SearchField.mdx

```diff
@@ -72,6 +73,24 @@ We include trailing children to allow for additional input context to be provide
 </FieldWrap>
 ```
 
+### Controllable
+
+```jsx example
+function ControllableSearchFieldExample() {
+  const [search, setSearch] = useState("");
+
+  return (
+    <FieldWrap>
+      <SearchField
+        label="Controllable"
+        value={search}
+        onChange={(value) => setSearch(value)}
+      />
+    </FieldWrap>
+  );
+}
```

Comment:
Just some minor nits to bring it in line with other component documentation:

```suggestion
  const [value, setValue] = useState('Banana');
  const handleChange = (value) => {
    setValue(value);
  };
  
  return (
    <FieldWrap>
      <SearchField
        label="Controlled"
        value={value}
        onChange={handleChange}
      />
    </FieldWrap>
  );
}
```

Reply:
Updated